plugins/generator-1.19.2/forge-1.19.2/templates/block/block.java.ftl

```diff
@@ -193,12 +193,21 @@ public class ${name}Block extends
    	}
 	</#if>
 
-	<#if data.specialInfo?has_content>
-	@Override public void appendHoverText(ItemStack itemstack, BlockGetter world, List<Component> list, TooltipFlag flag) {
-		super.appendHoverText(itemstack, world, list, flag);
-		<#list data.specialInfo as entry>
-		list.add(Component.literal("${JavaConventions.escapeStringForJava(entry)}"));
-	    </#list>
+	<#if data.specialInformation?has_content || hasProcedure(data.specialInformation)>

```

Comment:
As block and plant both seem to have the same approach, you could also move this to triggers and call it eg. `addSpecialInformationBlock`

Reply:
Done. Instead of making a whole new macro, I just added an `isBlock` boolean, to change the `Level` into a `BlockGetter` as it is the only difference between that, and other call such as Item.

Reply:
That is even better :)